public/js/positionList.js

```diff
@@ -0,0 +1,83 @@
+const openEditPosition = (id) => {
+  window.location.href = `${window.location.origin}/views/positionForm.html?_id=${id}`;
+};
+
+const openNewPosition = () => {
+  window.location.href = `${window.location.origin}/views/positionForm.html`;
+};
+
+window.onload = () => {
+  const navButton = document.getElementById('positionsNav');
+  navButton.classList.add('activePage');
+
+  const tableContent = document.getElementById('positions-table-content');
+
+  const addPosition = document.getElementById('addPosition');
+  addPosition.onclick = openNewPosition;
+
+  const modalSection = document.getElementById('modal-section');
+  const openModal = () => {
+    modalSection.classList.add('modal-section-display-on');
+  };
+
+  const closeModal = (e) => {
```

Comment:
move all the modal logic to a separate js file, please see how it was made [here](https://github.com/BaSD-2021/Esteban-MindSet-server/pull/45/files#diff-784398170010d72b231f1be9923608a49ab2b04a28fa56848def15e971a9283e)

Reply:
done!